### PR TITLE
Add bursary content back to landing pages

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/aFtKp6z8/6687

### CODE REVIEW

File: app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb [hunk 1]
```diff
@@ -46,7 +46,7 @@
     <h3 class="heading-l">Can I get help funding my training?</h3>
 
     <p>
-      If you're interested in teaching certain subjects, you might be able to get a tax-free bursary or scholarship, which doesn't need to be paid back, to support you while you are training.
+      If you're interested in teaching certain subjects, you might be able to get up to <%= v :scholarships_generic_maxshortened %> tax-free, which doesn't need to be paid back, to support you while you are training.
```
Comment (gemmadallmandfe) [inline app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb:49]: @emmaachesongray I know this is how we worded it in prod previously but it is worth making it slightly clearer?

```suggestion
      If you're interested in teaching certain subjects, you might be able to get a tax-free bursary or scholarship of up to <%= v :scholarships_generic_maxshortened %>, which doesn't need to be paid back, to support you while you are training.
```

Comment (gemmadallmandfe) [reply app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb:49]: (or use the wording on `/landing/how-to-become-a-teacher`)

File: app/views/content/landing/how-to-fund-your-teacher-training/_collage.html.erb [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <section class="col col-full-content">
       <div class="inset">
         <p class="col-720">
-          If you train to be a teacher, you could be eligible to receive a scholarship or bursary to help you train.
+          If you train to be a teacher, you could be eligible to receive a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> to help you train.
```
Comment (gemmadallmandfe) [inline app/views/content/landing/how-to-fund-your-teacher-training/_collage.html.erb:6]:
```suggestion
          If you train to be a teacher, you could be eligible to get a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> to help you train.
```

I know this was the prod wording previously - worth simplifying a bit?

Comment (emmaachesongray) [reply app/views/content/landing/how-to-fund-your-teacher-training/_collage.html.erb:6]:
```suggestion
          If you train to be a teacher, you could be eligible to receive a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> tax-free. This is to support you while you are training and does not need to be paid back.
```

File: app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb (hunk 1 above)
Comment (emmaachesongray) [reply app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb:49]:
```suggestion
      If you're interested in teaching certain subjects, you might be able to get up to <%= v :scholarships_generic_maxshortened %> tax-free. This is to support you while you're training and does not need to be paid back.
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Review by Sarah-DfE — APPROVED** [review]
Looks good to me

**Pull request merged**